samples/bookinfo/src/ratings/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM node:4-slim
+FROM node:12-slim
```

Comment:
Compared the other docker images used, this one doesn't seem very specific using `12`, where as others uses `8.0.6`,  `19.0.0.4`,  or `2.6.3` for example. Maybe make this `12.7.0`?